core/gui/src/app/hub/component/workflow/detail/hub-workflow-detail.component.ts

```diff
@@ -223,20 +225,13 @@ export class HubWorkflowDetailComponent implements AfterViewInit, OnDestroy, OnI
     }
   }
 
-  formatLikeCount(count: number): string {
+  formatCount(count: number): string {
     if (count >= 1000) {
       return (count / 1000).toFixed(1) + "k";
     }
     return count.toString();
   }
 
-  formatViewCount(count: number): string {
-    if (!this.displayPreciseViewCount && count >= 1000) {
-      return (count / 1000).toFixed(1) + "k";
-    }
-    return count.toString();
-  }
```

Comment:
are those changes related to this PR?

Reply:
Sorry, these are unrelated. I will open a new PR to remove these redundant pieces of code.

Reply:
in the future, please let the reviewer to resolve comments. we need to refer back to them when we review the PR again.